ambari-server/src/main/resources/stacks/BIGTOP/3.3.0/services/ZOOKEEPER/configuration/zoo.cfg.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+<!--
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+<configuration>
+   <property>
+    <name>admin.serverPort</name>
+    <value>8180</value>
+    <description>admin.serverPort</description>
+    <on-ambari-upgrade add="true"/>
+  </property>
+  <property>
+    <name>admin.enableServer</name>
+    <value>true</value>
+    <display-name>Enable Admin Server </display-name>
+    <description>Enable Admin Server.</description>
+    <value-attributes>
+      <type>boolean</type>
+    </value-attributes>
+    <on-ambari-upgrade add="true"/>
+  </property>
+  <property>
+    <name>4lw.commands.whitelist</name>
+    <value>ruok</value>
```

Comment:
Can we leave comment explaining why we need this?

Reply:
The redundant ZK configuration has been removed. The 4lw.commands.whitelist is already defined in the Bigtop 3.2.0 stack, so it will naturally be inherited in 3.3.0 without any additional definition required.